### PR TITLE
[ADD] sign: add a 1st doc for Sign on the validity of e-signatures

### PR DESCRIPTION
**Pull request by xpl-odoo**
As requested by CJA: https://www.odoo.com/web#active_id=2227553&cids=1&id=2227553&model=project.task&menu_id=
(to be added first to 13.0 as per ANV's instructions)

### CODE REVIEW

**Review by AntoineVDV — COMMENTED** [review]
Hi @xpl-odoo, I updated the label since we use "new content request" for issues (issues and PRs share their labels).
Apart from that, the RST is neat :ok_hand: 
For the commit message, you could just go with "[ADD] sign: document the validity of e-signatures"

File: sign/overview/signature_validity.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The legal validity of electronic signatures generated by Odoo depends on the legislation of your
+country. Companies doing business abroad should consider electronic signature laws of other
+countries as well.
+
+In the European Union
+=====================
+
+The `eIDAS regulation <http://data.europa.eu/eli/reg/2014/910/oj>`_ establishes the framework for
+electronic signatures in all `27 member states of the European Union
+<https://europa.eu/european-union/about-eu/countries_en>`_
```
Comment (jcs-odoo) [inline sign/overview/signature_validity.rst:14]:
```suggestion
The `eIDAS regulation <http://data.europa.eu/eli/reg/2014/910/oj>`_ establishes the framework for
electronic signatures in all `27 member states of the European Union
<https://europa.eu/european-union/about-eu/countries_en>`_.
```

File: sign/overview/signature_validity.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+===============================
+
+The `ESIGN Act (Electronic Signatures in Global and National Commerce Act)
+<https://www.fdic.gov/regulations/compliance/manual/10/X-3.1.pdf>`_, at the interstate and
+international levels, and `UETA (Uniform Electronic Transactions Act)
+<https://www.uniformlaws.org/committees/community-home/librarydocuments?communitykey=2c04b76c-2b7d-4399-977e-d5876ba7e034&tab=librarydocuments>`_,
+at the state level, provide the legal framework for electronic signatures. Note that `Illinois
+<https://www.ilga.gov/legislation/ilcs/ilcs5.asp?ActID=89&>`_ and `New York
+<https://its.ny.gov/electronic-signatures-and-records-act-esra>`_ have not adopted UETA, but similar
+acts instead.
```
Comment (jcs-odoo) [inline sign/overview/signature_validity.rst:40]:
```suggestion
The `ESIGN Act (Electronic Signatures in Global and National Commerce Act)
<https://www.fdic.gov/regulations/compliance/manual/10/X-3.1.pdf>`_, at the interstate and
international levels, and the `UETA (Uniform Electronic Transactions Act)
<https://www.uniformlaws.org/committees/community-home/librarydocuments?communitykey=2c04b76c-2b7d-4399-977e-d5876ba7e034&tab=librarydocuments>`_,
at the state level, provide the legal framework for electronic signatures. Note that `Illinois
<https://www.ilga.gov/legislation/ilcs/ilcs5.asp?ActID=89&>`_ and `New York
<https://its.ny.gov/electronic-signatures-and-records-act-esra>`_ have not adopted the UETA, but similar
acts instead.
```

File: sign/overview/signature_validity.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Overall, to be recognized as valid, electronic signatures have to meet five criteria:
+
+#. A signer must show a clear intent to sign. For example, using a mouse to draw a signature can
+   show intent. The signer must also have the option to opt-out of electronically signing a
+   document.
+#. A signer must first express or imply their consent to conduct business electronically.
+#. The signature must be clearly attributed. In Odoo, metadata, such as the signer’s IP address, is
+   added to the signature, which can be used as supporting evidence.
+#. The signature must be associated with the document being signed, for example, by keeping a record
+   that details how the signature was captured.
```
Comment (jcs-odoo) [inline sign/overview/signature_validity.rst:51]:
```suggestion
#. The signature must be associated with the document being signed, for example, by keeping a record
   detailing how the signature was captured.
```

File: sign/overview/signature_validity.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+#. Electronically signed documents need to be retained and available for later reference by all
+   parties involved, for example by providing the signer either a fully-executed copy or the option
+   to download a copy.
```
Comment (jcs-odoo) [inline sign/overview/signature_validity.rst:54]:
```suggestion
#. Electronically signed documents need to be retained and available for later reference by all
   parties involved, for example, by providing the signer either a fully-executed copy or the option
   to download a copy.
```

File: sign/overview/signature_validity.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+.. note::
+   The information provided here does not constitute legal advice; it is provided for general
+   informational purposes only. As laws governing electronic signatures evolve rapidly, we cannot
+   guarantee that the information is current. We advise you to should contact a local attorney to
```
Comment (jcs-odoo) [inline sign/overview/signature_validity.rst:59]:
```suggestion
   guarantee that the information is up to date. We advise you to should contact a local attorney to
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @xpl-odoo,

good job!

I just need a few small changes (if you agree with my suggestions) + renaming the commit as Antoine suggested above (this can be done when you do the `git commit --amend` command), then I'll approve it.

Cheers,

**Comment by xpl-odoo** [comment]
I have 'accepted' all your suggestions and renamed the commit accordingly.

**Review by jcs-odoo — APPROVED** [review]

**Pull request merged**